mkdocs.yml

```diff
@@ -242,29 +238,26 @@ nav:
-                    - 'Basket routing': 'guide/basket/basket_api/basket_routing.md'
+                - 'Basket data model': 'guide/basket/basket_api/basket_data_model.md'
+                - 'BasketService': 'guide/basket/basket_api/basketservice.md'
+                - 'Basket Events': 'guide/basket/basket_api/basket_events.md'
+                - 'Basket routing': 'guide/basket/basket_api/basket_routing.md'
             - 'Wishlist and stored baskets': 'guide/basket/wishlist_and_stored_baskets.md'
             - Checkout:
                 - 'Checkout': 'guide/checkout/checkout.md'
                 - 'Checkout templates': 'guide/checkout/checkout_templates.md'
                 - 'Order confirmation': 'guide/checkout/order_confirmation.md'
                 - 'Local orders': 'guide/checkout/local_orders.md'
                 - 'Return process': 'guide/checkout/return_process.md'
-                - Checkout API:
-                    - 'Checkout controllers': 'guide/checkout/checkout_api/checkout_controllers.md'
-                    - 'Checkout events': 'guide/checkout/checkout_api/checkout_events.md'
-                    - Forms:
-                        - 'Configuration for checkout forms': 'guide/checkout/checkout_api/forms/configuration_for_checkout_forms.md'
-                        - 'Invoice address form': 'guide/checkout/checkout_api/forms/checkout_invoice_address_form.md'
-                        - 'Delivery address form': 'guide/checkout/checkout_api/forms/checkout_delivery_address_form.md'
-                        - 'Shipping payment form': 'guide/checkout/checkout_api/forms/checkout_shipping_payment_form.md'
-                        - 'Summary form': 'guide/checkout/checkout_api/forms/checkout_summary_form.md'
-                        - 'Order summary': 'guide/checkout/checkout_api/forms/order_summary.md'
-                        - 'Services for checkout forms': 'guide/checkout/checkout_api/forms/services_for_checkout_forms.md'
+                - 'Checkout controllers': 'guide/checkout/checkout_api/checkout_controllers.md'
+                - 'Checkout events': 'guide/checkout/checkout_api/checkout_events.md'
+                - 'Configuration for checkout forms': 'guide/checkout/checkout_api/forms/configuration_for_checkout_forms.md'
+                - 'Invoice address form': 'guide/checkout/checkout_api/forms/checkout_invoice_address_form.md'
+                - 'Delivery address form': 'guide/checkout/checkout_api/forms/checkout_delivery_address_form.md'
+                - 'Shipping payment form': 'guide/checkout/checkout_api/forms/checkout_shipping_payment_form.md'
+                - 'Summary form': 'guide/checkout/checkout_api/forms/checkout_summary_form.md'
+                - 'Order summary': 'guide/checkout/checkout_api/forms/order_summary.md'
+                - 'Services for checkout forms': 'guide/checkout/checkout_api/forms/services_for_checkout_forms.md'
```

Comment:
I think I would move this line up, and place it right after "Configuration for checkout forms"